src/pages/LikedFoodPage.js

```diff
@@ -3,27 +3,26 @@ import Card from '../components/Card.js';
 import Navigation from '../components/Navigation.js';
 
 export default function LikedFoodPage({ storageData }) {
-  const filteredLikedData = storageData?.filter(
-    data => data.foodJudge === 'liked'
-  );
+  const likedData = storageData?.filter(data => data.foodJudge === 'liked');
   return (
     <LikedFoodPageStyle>
       <Header>
         <h1>Mir schmeckt...</h1>
       </Header>
       <main>
-        {filteredLikedData.length === 0 ? (
+        {likedData.length === 0 ? (
           <p>
-            Du hast hier noch keine Listeneinträge. Listeneinträge erscheinen,
-            wenn das Formular ausgefüllt und gespeichert wird.
+            Du hast hier noch keine Listeneinträge. <br />
+            Listeneinträge erscheinen, wenn das Formular ausgefüllt und
+            gespeichert wird.
```

Comment:
good work so far...for an user it could be nice to have an link here where the user would be directed to the adding page! but it´s not necessary

Reply:
thanks for considering this, i also think, that it is not necessary here as the user has the navigation bar to switch to the form page with one click